libs/product/src/models/composite-product.ts

```diff
@@ -0,0 +1,12 @@
+import { DaffProduct } from './product';
+import { DaffCompositeProductItem } from './composite-product-item';
+
+export enum DaffCompositeItemsEnum {
+	together = 'TOGETHER',
+	separately = 'SEPARATELY'
+}
+
+export interface DaffCompositeProduct extends DaffProduct {
+	ship_bundle_items: DaffCompositeItemsEnum;
```

Comment:
We don't have a use for this, let's remove it.

Reply:
removed